config/locales/diaspora/en.yml

```diff
@@ -427,9 +427,9 @@ en:
       restrict_posts_i_see_q: "Can I restrict the posts in my stream to just those from certain aspects?"
       restrict_posts_i_see_a: "Yes. Click on “My aspects” in the side-bar and then click individual aspects in the list to select or deselect them. Only the posts by people in the selected aspects will appear in your stream."
       rename_aspect_q: "Can I rename an aspect?"
-      rename_aspect_a: "Yes. In your list of aspects on the left side of the main page, point your mouse at the aspect you want to rename. Click the little “Edit” pencil that appears to the right. You will be taken to the page for this aspect. Click the edit pencil next to the aspect name at the top of this page, the change the name and press “Update”."
+      rename_aspect_a: "Yes. Go to the “Contacts” page form the user menu and select the aspect you want to rename. You can also go there from the stream : click on “My aspects” on the left side of the main page, point your mouse at the aspect you want to rename and click the little “Edit” pencil that appears to the right. You will be taken to the page for this aspect. Click the edit pencil next to the aspect name at the top of this page, then change the name and press “Update”."
```

Comment:
I don't understand your point. You have to click the pencil in the sidebar, and again on the aspect page. If you click the aspect name in the sidebar, it selects/deselects that aspect in the stream, it doesn't help you to rename the aspect.